nixos/modules/services/monitoring/mimir.nix

```diff
@@ -32,11 +32,21 @@ in {
       type = types.package;
       description = lib.mdDoc ''Mimir package to use.'';
     };
+
+    extraFlags = mkOption {
+      type = types.listOf types.str;
+      default = [];
+      example = [ "--config.expand-env=true" ];
+      description = lib.mdDoc ''
+        Specify a list of additional command line flags,
+        which get escaped and are then passed to Mimir.
+      '';
+    };
   };
 
   config = mkIf cfg.enable {
     # for mimirtool
-    environment.systemPackages = [ pkgs.mimir ];
+    environment.systemPackages = [ cfg.package ];
```

Comment:
This was forgotten to adjust in 3cb385953bcdd0dc8676efd5dd28c520457ca549, and probably doesn't warrant a separate PR (but perhaps a separate commit?)

Reply:
IMO it's fine to leave it in the same commit.